resim/testing/fuzz_helpers.hh

```diff
@@ -86,4 +86,19 @@ Int random_element(TypeTag<Int> /*unused*/, InOut<Rng> rng) {
   return dist(*rng);
 }
 
+// random_element overload with an implementation for floating point types. This
+// is done here so it is in the same namespace as TypeTag and can be found by
+// ADL
+// when called in generic code.
+template <std::floating_point Float, typename Rng>
+Float random_element(TypeTag<Float> /*unused*/, InOut<Rng> rng) {
+  std::uniform_real_distribution<Float> dist{
+      std::numeric_limits<Float>::min(),
```

Comment:
I try to avoid this like the plague, for reasons documented here: https://en.cppreference.com/w/cpp/types/numeric_limits/min. Not sure if it will break things in any tests here, but I'd suggest `std::numeric_limits<Float>::lowest()`

Reply:
I've changed this to include negative values. However, I just learned that max() - lowest() doesn't work for [this reason](https://stackoverflow.com/questions/66854143/stduniform-real-distribution-always-returns-infinity).

Reply:
Truly fascinating. Probably good we stay away from double max anyway, so I think this implementation is better regardless...